Doc/library/logging.rst

```diff
@@ -1034,12 +1034,12 @@ interchangeably.
    to :class:`LoggerAdapter`.  These methods delegate to the underlying logger.
 
 .. versionchanged:: 3.6
-   Attribute :attr:`manager` and method :meth:`_log` were added, which
+   Attribute :attr:`!manager` and method :meth:`!_log` were added, which
```

Comment:
Perhaps these should be documented with:

```rst
   .. attribute:: manager

   Delegates to the underlying :attr:`!manager`` on the *logger*.

   .. attribute:: _log

   Delegates to the underlying :meth:`!_log`` method on the *logger*.
 ```

Reply:
Yes; let's do that first.

Reply:
gh-108145